spyder_kernels/customize/spydercustomize.py

```diff
@@ -209,6 +209,22 @@ def __init__(self, *args, **kwargs):
 except:
     pass
 
+#==============================================================================
+# Turtle adjustments
+#==============================================================================
+# Needed to prevent turtle scripts crashes after multiple launches in the same
+# IPython Console instance. See Spyder issue #6278
```

Comment:
Please change this to

```python
# This is needed to prevent turtle scripts crashes after multiple runs in the same
# IPython Console instance.
# See Spyder issue #6278
```